### DISCUSSION TITLE
The Recipes never show on the admin->configuration->Recipes

### DISCUSSION THREAD

**Discussion by liusming** [OrchardCMS/OrchardCore#11101] [Q&A] [answered]
I have set up Orchard 1.0 and 1.2.2 and in the https://try.orchardcore.net/ but fond the same issues. The list under admin->configuration->Recipes always shows "Nothing here! There are no recipes for the moment.". 
![Screen Shot 2022-01-31 at 12 54 09 PM](https://user-images.githubusercontent.com/4685718/151741286-976ef477-4aa7-4ab3-9bc5-0a8d4e661297.png)
I have try to export some json file from Deployment Plans then import but still nothing show on the list. Unlike the demo movies on the web with RC2 their show be all the recipes after orchard installed. Is there anything missing for my installation? Please kindly advise.
![Screen Shot 2022-01-31 at 1 06 07 PM](https://user-images.githubusercontent.com/4685718/151741748-a8fa527d-b179-45ef-9b4f-064395d8a59b.png)

**Comment by Skrypt** [comment, marked as answer]
There are "setup" recipes that are included in an Orchard Core Theme. There are deployment recipes that can be imported or deployed with the import/export feature. And there are recipes that can be added to modules so that they can be shared in the Recipe List that you are mentioning. You can see an example of that in the OrchardCore.Commerce module here : https://github.com/OrchardCMS/OrchardCore.Commerce/tree/master/OrchardCore.Commerce/Recipes

So, of course, it won't show recipes that you used to import content in Orchard Core. The import recipe form imports content and not recipes ...

**Reply by liusming** [reply]
Thanks a lot. I will try to build a module to test.
